### PR TITLE
Bugfix multipolygon treatment and adapt to latest Shapely

### PR DESCRIPTION
**Pull request by adybbroe**
This PR mainly fixes a problem in the treatment of multi polygons for newer shapely versions.
For a while this deprecation warning has been seen.
```
ShapelyDeprecationWarning: Iteration over multi-part geometries is deprecated and will be removed in Shapely 2.0. Use the `geoms` property to access the constituent parts of a multi-part geometry.
```

Now with shapely 2.0 it results in a `TypeError` and a failure.

 - [ ] Closes #xxxx 
 - [x] Tests added 
 - [x] Tests passed: Passes ``pytest`` 
 - [x] Passes ``flake8`` 
 - [ ] Fully documented

### CODE REVIEW

File: activefires_pp/post_processing.py [hunk 1]
```diff
@@ -329,7 +329,9 @@ def get_global_mask_from_shapefile(shapefile, lonlats, start_geom_index=0):
     shape = geometry.geoms[0]
     pth = Path(shape.exterior.coords)
     mask = pth.contains_points(points)
-    for shape in geometry.geoms[start_geom_index:]:
+
+    constituent_part = geometry.geoms[start_geom_index:]
+    for shape in constituent_part.geoms:
         pth = Path(shape.exterior.coords)
         mask = np.logical_or(mask, pth.contains_points(points))
```
Comment (mraspaud) [inline activefires_pp/post_processing.py:337]: this looks very similar to `get_maks_from_mulitpolygon`, can it be factorized?

**Review by mraspaud — COMMENTED** [review]
Just one refactor proposition, otherwise lgtm

File: activefires_pp/post_processing.py (hunk 1 above)
Comment (adybbroe) [reply activefires_pp/post_processing.py]: Yes, true. I think this should be done okay now, and at least one test is covering this

**Review by mraspaud — APPROVED** [review]
LGTM

**Pull request merged**